File: tasks/成果レポート自動化/extract_unique.py
import json

with open('/tmp/a_column.json', encoding='utf-8') as f:
    data = json.load(f)

values = [row[0] for row in data.get('values', []) if row]
unique = sorted(set(values))

with open('/tmp/unique_names.txt', 'w', encoding='utf-8') as out:
    for v in unique:
        out.write(v + '\n')

print(f"Found {len(unique)} unique names, saved to /tmp/unique_names.txt")

File: tasks/成果レポート自動化/extract_unique.js
const fs = require('fs');
const data = JSON.parse(fs.readFileSync('/tmp/a_column.json', 'utf-8'));
const values = data.values.filter(r => r.length > 0).map(r => r[0]);
const unique = [...new Set(values)].sort();
unique.forEach(v => console.log(v));
